prime-dotnet-webapi/Configuration/AccessTerms/AgreementConfiguration.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+using Prime.Models;
+using Prime.Configuration.Resources;
+
+namespace Prime.Configuration.Agreements
+{
+    public static class AgreementConfiguration
+    {
+        private static DateTime SEEDING_DATE = SeededTable<Agreement>.SEEDING_DATE;
+
+        public static IEnumerable<Agreement> SeedData
+        {
+            get
+            {
+                return new Agreement[]{
+                    new OboAgreement                 { Id = 1, Text = "obo-access-terms-v1.html", EffectiveDate = SEEDING_DATE, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new RegulatedUserAgreement       { Id = 2, Text =  "ru-access-terms-v1.html", EffectiveDate = SEEDING_DATE, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new OboAgreement                 { Id = 3, Text = "obo-access-terms-v2.html", EffectiveDate = DateTimeOffset.Parse("2020-03-05 00:00:00"), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new RegulatedUserAgreement       { Id = 4, Text =  "ru-access-terms-v2.html", EffectiveDate = DateTimeOffset.Parse("2020-03-05 00:00:00"), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new OboAgreement                 { Id = 5, Text = "obo-access-terms-v3.html", EffectiveDate = DateTimeOffset.Parse("2020-03-10 00:00:00"), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new RegulatedUserAgreement       { Id = 6, Text =  "ru-access-terms-v3.html", EffectiveDate = DateTimeOffset.Parse("2020-05-07 00:00:00"), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new OboAgreement                 { Id = 7, Text = "obo-access-terms-v4.html", EffectiveDate = DateTimeOffset.Parse("2020-05-07 00:00:00"), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new RegulatedUserAgreement       { Id = 8, Text =  "ru-access-terms-v4.html", EffectiveDate = DateTimeOffset.Parse("2020-06-03 00:00:00"), CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new CommunityPharmacistAgreement { Id = 9, Text =  "com-pharm-terms-v1.html", EffectiveDate = SEEDING_DATE, CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
```

Comment:
Seed data has the filename as placeholder text now. This should let us use the agreements in tests without worrying about huge amounts of string data. The Configure() methods for each type of agreement call LoadText() when configuring the database to load the text from the file when needed.